sql/parser/alter_test.go

```diff
@@ -34,3 +36,59 @@ func TestParserAlterTable(t *testing.T) {
 		})
 	}
 }
+
+func TestParserAlterTableAddColumn(t *testing.T) {
+	tests := []struct {
+		name     string
+		s        string
+		expected query.Statement
+		errored  bool
+	}{
+		{"Basic", "ALTER TABLE foo ADD FIELD bar", query.AlterTableAddField{TableName: "foo",
+			Constraint: database.FieldConstraint{
+				Path: parsePath(t, "bar"),
+			},
+		}, false},
+		{"With type", "ALTER TABLE foo ADD FIELD bar integer", query.AlterTableAddField{TableName: "foo",
+			Constraint: database.FieldConstraint{
+				Path: parsePath(t, "bar"),
+				Type: document.IntegerValue,
+			},
+		}, false},
+		{"With not null", "ALTER TABLE foo ADD FIELD bar NOT NULL", query.AlterTableAddField{TableName: "foo",
+			Constraint: database.FieldConstraint{
+				Path:      parsePath(t, "bar"),
+				IsNotNull: true,
+			},
+		}, false},
+		{"With primary key", "ALTER TABLE foo ADD FIELD bar PRIMARY KEY", query.AlterTableAddField{TableName: "foo",
```

Comment:
A `PRIMARY KEY` field should hold unique value therefore we shouldn't support altering a table by adding a `PRIMARY KEY` constraint. We don't have a way to update all existing documents with different values.

Reply:
Good catch @yaziine! Indeed, we don't have a strategy for changing the primary key yet. We should not allow this for now